src/codeu/chat/common/BasicController.java

```diff
@@ -31,6 +31,11 @@
   //   state of the message on the server.
   Message newMessage(Uuid author, Uuid conversation, String body);
 
+  // DELETE MESSAGE
+  //
+  //   Delete a message from the server.
+  boolean deleteMessage(Uuid msg, Uuid conversation);
```

Comment:
I don't think this belongs in the controller because the controller is read only. This should only be in Chat.java and the model that goes with Chat.java.